docs/_docs/getting-started/openidconnect-configuration.md

```diff
@@ -170,3 +187,31 @@ $ curl https://auth.example.com/auth/realms/example/protocol/openid-connect/user
 
 > Dependency-Track associates every OpenID Connect user with their subject identifier (`sub` claim of the access token) upon first login.
 > If a user with the same name but a different subject identifier attempts to log in via OIDC, Dependency-Track will refuse to authenticate that user. This is done to prevent account takeovers, as some identity providers allow users to change their usernames. Also, uniqueness of usernames is not always guaranteed, while the uniqueness of subject identifiers is.
+
+### Azure Active Directory app registration
+
+The following steps demonstrate how to setup OpenID Connect with Azure Active Directory.
+
+> This guide assumes that:
+>   * the Dependency-Track frontend has been deployed to `https://dependencytrack.example.com`
+>   * an Azure Active Directory tenant has been created with name `dt-example`
```

Comment:
Is the name `dt-example` important? It isn't mentioned anywhere in the following steps.

```suggestion
>   * an Azure Active Directory tenant has been created
```